### PR TITLE
PRODSEC-1262 Whitesource Transition to Snyk

### PR DESCRIPTION
**Pull request by sfc-gh-jfan**

### CODE REVIEW

**Comment by sfc-gh-bli** [comment]
@sfc-gh-jfan why request us to review it?

**Comment by sfc-gh-mrui** [comment]
@sfc-gh-jfan It looks the github action is `skip`. Is this expected?
Should we remove the usage for white source?

**Comment by sfc-gh-jfan** [comment]
> @sfc-gh-jfan why request us to review it?

Hey, @sfc-gh-bli  I didn't add anyone myself into it. seems you are defined as code owner, thus need your review before merge. Lmk if someone else should review that, thanks!

**Comment by sfc-gh-jfan** [comment]
> @sfc-gh-jfan It looks the github action is `skip`. Is this expected? Should we remove the usage for white source?

Hey, @sfc-gh-mrui yes, it is expected since PR has not yet merged. And yes, let us remove the usage for whitesource, I have removed .whitesource file. (I think we should remove whitesource folder as well? since it might include something else, so I didn't remove it in this PR)

File: deploy.sh
Comment (sfc-gh-mrui) [inline deploy.sh:101]: There are another 2 places to call `run_whitesource.sh`, they should be removed too.

Comment (sfc-gh-jfan) [reply deploy.sh:101]: good catch, thanks! just pushed an update

**Review by sfc-gh-mrui — APPROVED** [review]
Thanks. LGTM

**Review by sfc-gh-bli — APPROVED** [review]
LGTM

**Pull request merged**
